tests/src/test/java/org/mozilla/javascript/tests/harmony/GeneratorsYieldStarReturnTest.java

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.javascript.tests.harmony;
+
+import org.mozilla.javascript.Context;
+import org.mozilla.javascript.drivers.LanguageVersion;
+import org.mozilla.javascript.drivers.RhinoTest;
+import org.mozilla.javascript.drivers.ScriptTestsBase;
+
+@RhinoTest("testsrc/jstests/harmony/generators-yield-star-return.js")
+@LanguageVersion(Context.VERSION_ES6)
+public class GeneratorsYieldStarReturnTest extends ScriptTestsBase {}
```

Comment:
personally i like the plain java test cases more - it is much easier to debug a single case and it is more obvious to point to the failing function. On the other hand the spotless re-formatting of the code is sometimes a pain.

Reply:
Makes sense, let me try to redo them as plain junit tests.